toxicity/src/index.ts

```diff
@@ -52,7 +49,9 @@ export class ToxicityClassifier {
   }
 
   async loadModel() {
-    return tfconv.loadGraphModel(`${BASE_PATH}model.json`);
+    return tfconv.loadGraphModel(
+        'https://tfhub.dev/tensorflow/tfjs-model/toxicity/1/default/1',
```

Comment:
Are you loading the original model without training for the particular task?

Reply:
It's the same model we've been serving, except now the assets live on TFHub.